docs/getting_started/execution-modes.rst

```diff
@@ -148,3 +153,29 @@ Example DAG:
             "secrets": [postgres_password_secret],
         },
     )
+
+Azure Container Instance
+------------------------
+Similar to the ``kubernetes`` approach, using ``Azure Container Instances`` as the execution mode gives a very isolated way of running ``dbt``, since the ``dbt`` run itself is run within a container running in an Azure Container Instance.
```

Comment:
```suggestion
Azure Container Instance
------------------------
.. versionadded:: 1.4
Similar to the ``kubernetes`` approach, using ``Azure Container Instances`` as the execution mode gives a very isolated way of running ``dbt``, since the ``dbt`` run itself is run within a container running in an Azure Container Instance.
```

Reply:
Done @jbandoro!